lib/matplotlib/tight_layout.py

```diff
@@ -36,7 +36,8 @@ def _get_top(tight_bbox, axes_bbox):
 
 def auto_adjust_subplotpars(
         fig, renderer, nrows_ncols, num1num2_list, subplot_list,
-        ax_bbox_list=None, pad=1.08, h_pad=None, w_pad=None, rect=None):
+        ax_bbox_list=None, pad=rcParams['figure.tight_layout.pad'],
+        h_pad=rcParams['figure.tight_layout.h_pad'], w_pad=rcParams['figure.tight_layout.w_pad'], rect=None):
```

Comment:
Note that it's quite uncommon to have the rcParams being evaluated within the function signature. I think this would at least become a problem with the rc_context manager. Better keep the paramers as `None` and set the values within the function (`if param is None: ...`).

Reply:
In fact, this is not just uncommon, but it will not work as expected. The rcParams would be resolved when the function is read from the file but the intent is for them to be resolved when the function is called.